wolfcrypt/src/wc_port.c

```diff
@@ -1175,6 +1175,19 @@ int wolfSSL_CryptHwMutexUnLock(void) {
         return 0;
     }
 
+    int ebsnet_fseek(int a, long b, int c)
```

Comment:
This function will need to be defined in a header... ideally wc_port.h above the `#define XFSEEK                   ebsnet_fseek`

Reply:
I'm highly confident this is prototyped in vfile.h (the EBSnet file system API header) but I am happy to confirm that with the customer.

Reply:
@dgarske I'll CC you on the email.